### PR TITLE
WRR-12612:  Enactjs docs website shows wrong library logos

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed docs to show correct logo for each library in API page

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fixed docs to show correct logo for each library in API page

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
No prior PR introduced this bug, GraphQL queries are inconsistent.

### Links
[//]: # (Related issues, references)
WRR-12612

### Comments
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by seunghoh — CHANGES_REQUESTED** [review]
Thanks for fixing the issue but, I have some requests to update.
- Would you please update PR description in detail? I wonder why this regression occurred and which ticket. 
- The enactjs.com for sandstone doc also had the moonstone. I don't know when it disappeared.

File: src/pages/docs/modules/index.js [hunk 1]
```diff
@@ -31,15 +31,24 @@ const Doc = class ReduxDocList extends Component {
...
 		const packageImages = {
-			core: imagesFromProps[0].node.publicURL,
-			i18n: imagesFromProps[1].node.publicURL,
-			moonstone: imagesFromProps[2].node.publicURL,
-			spotlight: imagesFromProps[3].node.publicURL,
-			ui: imagesFromProps[4].node.publicURL,
-			webos: imagesFromProps[5].node.publicURL
+			core: imagesFromProps[coreLogoIndex],
+			i18n: imagesFromProps[i18nLogoIndex],
+			moonstone: imagesFromProps[moonstoneLogoIndex],
```
Comment (seunghoh) [inline src/pages/docs/modules/index.js:48]: Why is this `moonstone` still here? `moonstone` is not in the mono repo and SVG moved on https://github.com/enactjs/moonstone/pull/189

**Comment by seunghoh** [comment]
@adrian-cocoara-lgp Would you please revert the last commit which removed moonstone svg?

**Review by seunghoh — APPROVED** [review]

**Pull request merged**